### PR TITLE
allow deposit memo in deposits

### PR DESCRIPTION
**Pull request by acharb**
[ticket](https://stellarorg.atlassian.net/browse/WAL-912?atlOrigin=eyJpIjoiMzYwZGEwYjI1MDk3NDIwOGE3YmNkNTg4OTEyNjZiOTYiLCJwIjoiaiJ9)

realized missing when updating stellar docs in https://github.com/stellar/stellar-docs/pull/196

### CODE REVIEW

File: src/walletSdk/Anchor/Sep24.ts [hunk 1]
```diff
@@ -46,13 +46,15 @@ export class Sep24 {
...
     extraFields,
+    destinationMemo,
     destinationAccount,
   }: Sep24PostParams): Promise<Sep24PostResponse> {
     return this.flow({
       assetCode,
       authToken,
       lang,
       extraFields,
+      destinationMemo,
```
Comment (CassioMG) [inline src/walletSdk/Anchor/Sep24.ts:57]: Nice addition. Should we also add memo to `withdraw()` ? According to the SEP it should be possible: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#withdraw

Comment (acharb) [reply src/walletSdk/Anchor/Sep24.ts:57]: Ya i noticed that too but the Kotlin sdk didn't have it so I left it off

@Ifropc any reason for that? or can I go ahead and memos for withdrawals too

Comment (Ifropc) [reply src/walletSdk/Anchor/Sep24.ts:57]: It's deprecated and should be derived from SEP-10 token instead. I don't know of any anchor that doesn't support this functionality

**Review by Ifropc — APPROVED** [review]

File: src/walletSdk/Anchor/Sep24.ts (hunk 1 above)
Comment (CassioMG) [reply src/walletSdk/Anchor/Sep24.ts:57]: Makes sense, I've only seen cases for deposit memo

**Review by CassioMG — APPROVED** [review]

**Pull request merged**